doc/nrf/libraries/modem/modem_jwt.rst

```diff
@@ -9,32 +9,47 @@ Modem JWT
 
 The Modem JWT library provides access to the modem's `JSON Web Token (JWT)`_ generation feature, which is available in modem firmware v1.3.0 or higher.
 
+Configuration
+*************
+
 To use the library to request a JWT, complete the following steps:
 
-* Enable the library.
-* Populate the :c:struct:`jwt_data` structure with your desired values.
-* Pass the structure to the function that generates JWT (:c:func:`modem_jwt_generate`).
+1. Enable the library by setting the following Kconfig options:
```

Comment:
```suggestion
1. Set the following Kconfig options to enable the library:
```
I always want to desperately avoid the structure "Do something by ...ing something." Up to you, of course.